src/bc_common.d

```diff
@@ -102,12 +102,18 @@ enum BCTypeEnum : ubyte
 
 }
 
+enum BCTypeFlags
+{
+  Const = 0x1,
+}
 struct BCType
 {
     BCTypeEnum type;
     alias type this;
     /// 0 means basic type
     uint typeIndex;
+    // additional information
+    uint flags;
```

Comment:
That would later require to transiently apply flags to element types, e.g. to support `const(T*)` and `const(T)*` at the same time.
Not sure why the interpreter has to deal with const though.